### PR TITLE
Update pt_BR translations

### PR DESCRIPTION
**Pull request by aleprovencio**
As suggested on #1449

### CODE REVIEW

File: po/pt_BR.po [hunk 1]
```diff
@@ -1,20 +1,21 @@
 # Adiel Mittmann <adiel@inf.ufsc.br>, 2007, 2008, 2009, 2010.
 # Alexandre Erwin Ittner <alexandre@ittner.com.br>, 2011.
+# Alexandre Provencio <dev@aleprovencio.net>, 2011.
```
Comment (Minoru) [inline po/pt_BR.po:3]:
```suggestion
# Alexandre Provencio <dev@aleprovencio.net>, 2022.
```

File: po/pt_BR.po [hunk 2]
```diff
@@ -122,19 +123,16 @@ msgid "remove unreferenced items from cache"
...
-#, fuzzy
 msgid "configuration"
-msgstr "Carregando configuração..."
+msgstr "configuração"
 
 #. / This is printed out by --help before the path to the urls file
 #: newsboat.cpp:132
-#, fuzzy
 msgid "feed URLs"
 msgstr "URLs"
```
Comment (Minoru) [inline po/pt_BR.po:137]: I think "feed" wasn't translated here.

File: po/pt_BR.po [hunk 3]
```diff
@@ -1922,7 +1919,7 @@ msgstr "versão de Atom inválida"
 
 #: rss/parser.cpp:367
 msgid "no Atom version"
-msgstr "Atom sem versão"
+msgstr "sem versão ATOM"
```
Comment (Minoru) [inline po/pt_BR.po:1922]:
```suggestion
msgstr "sem versão Atom"
```

"RSS" is an abbreviation, but "Atom" is not, so no need to make all the letters uppercase.

**Review by Minoru — CHANGES_REQUESTED** [review]
Thank you! And thanks especially for updating some other translations.

I spotted a couple mistakes though, please take a look below.

File: po/pt_BR.po (hunk 2 above)
Comment (aleprovencio) [reply po/pt_BR.po:137]: Right, following the current translations the msgstr should be 'URLs da fonte'

File: po/pt_BR.po (hunk 3 above)
Comment (aleprovencio) [reply po/pt_BR.po:1922]: OK

**Review by Minoru — APPROVED** [review]
Yup, looks good, thank you @aleprovencio!

**Pull request merged**